syntax = 'proto3';

service RSSAggregator {
  rpc GetAll(Empty) returns (GetAllResponse) {}
  rpc Get(GetByIdRequest) returns (GetByIdResponse) {}
}

message NewsItem {
  string id = 1;
  string title = 2;
  string description = 3;
}

message Empty {}
message GetAllResponse { NewsItem item = 1; }
message GetByIdRequest { string id = 1; }
message GetByIdResponse { NewsItem item = 1; }